### ISSUE TITLE
Would you please upload a copy of the database?

### ISSUE THREAD

**Issue by cdkth** [saurass/College-Work-Automation#1]
Hello,

I would like to use this software. But I am getting errors when trying to open the attendance. 

`SQLSTATE[42S02]: Base table or view not found: 1146 Table 'college_work_automation.facultylogins' doesn't exist (SQL: select `category`, `branch` from `facultylogins` where `userid` = admin limit 1)`

It would be very helpful if you could upload a copy of the database. Please let me know how can I overcome this error.

Thank you,
Jenson

**Comment by saurass** [comment]
yeah sure, I will upload a copy of SQL file or write the migration and commit.
